src/plugins/factoids/storage.internal.js

```diff
@@ -0,0 +1,171 @@
+/* eslint-disable no-underscore-dangle */
+const fs = require('fs');
+const path = require('path');
+const { promisify, inspect } = require('util');
+
+const readFile = promisify(fs.readFile);
+const writeFile = promisify(fs.writeFile);
+
+const FILE_PATH = path.join(__dirname, 'factoids.json');
+
+function toKey(key) {
+  return (key && key.toLowerCase()) || null;
+}
+
+class Store {
+  constructor() {
+    this._items = new Map();
+    this._loaded = false;
```

Comment:
node has private fields, why use any underscores?

Reply:
It's currently running in an older version, but I'll try an upgrade to 12.x and use private fields if that works.